README.md

```diff
@@ -47,6 +47,20 @@ yarn install
 bundle exec rake
 ```
 
+#### Running JavaScript tests only
+
+Running [Jasmine][] tests in a web browser
+
+```
+rake jasmine
+```
+
+Running [Jasmine][] tests in CI
```

Comment:
Might be better to phrase this as run in console, as we use this task outside of CI too.